src/components/CalendarDatePicker/index.js

```diff
@@ -6,7 +6,7 @@ import PropTypes from 'prop-types';
 // import { LabelText } from './InputDateTime.styles';
 import 'react-datepicker/dist/react-datepicker.css';
 
-const CustomField = ({ value, onClick }) => (
+const CustomField = ({ value, onClick, error }) => (
```

Comment:
for what will be using this "error" prop?

Reply:
If the user clicks on save and doesn't input a value in this field, no?